### DISCUSSION TITLE
5차 데모발표 후 어떤 이야기를 할지

### DISCUSSION THREAD

**Discussion by yeopto** [woowacourse-teams/2023-haru-study#563] [아이디어]
코치들의 피드백 후 어떤 이야기들을 하면 좋을지 이야기해봅시다.

**Comment by woo-jk** [comment]
SSE vs Polling

**Comment by jaehee329** [comment]
1. SSE를 정말로 하나?
2. 4주 남았는데 주 단위로 큼직한 일정 추산
---
개인 의견으로는 polling 해보고 sse 하는 방법은 시간이 너무 없을 듯.
백엔드 자체적으로 해야 큰 과제가 많아서 둘 다 해보는 것은 어려워보여요
sse 하게 되면 네트워크 유실로 생길 수 있는 부분들 얘기해보죠~

**Comment by nlom0218** [comment]
## 큰 덩어리 + 아직 정답이 정해져 있지 않는 부분

하루스터디의 핵심 기능 - 함께 스터디 하기를 어떻게 할까?

## 큰 덩어리

반응형

## 작은 덩어리

기록 페이지이 데이터 순서 및 페이지네이션
기록페이지 캐시

**Comment by yeopto** [comment]
기획적으로 정해진게 있는데, 오늘 솔라의 피드백에 대해서 한번 이야기해보는 것도 좋은 것 같아요~

**Comment by aak2075** [comment]
그냥 SSE 버리고 사용자 유치, 성능 개선에 더 집중하면 어떨까 이야기 해보는 것도 좋을 것 같아요~

적절하지 않은 케이스에 사용하는 SSE를 적용하는 것보다 지금 모각코 컨셉으로 나가고 다른 부분에서 어필을 하는게 나을수도 있다는 생각입니다. 면접때 이야기할거라면 구멍이 숭숭 날 것 같은데 그걸 또 방어하기 위해서는 수 많은 리소스가 들어가고 확실하지 않을 것 같습니다

**Comment by woosung1223** [comment]
### 사용자 유치 계획
커뮤니티에 적극적으로 홍보하는 것 뿐만 아니라 직접 찾아다니면서 홍보를 해도 좋을듯 함

### SSE
일단은 SSE 구현을 도전해보는 건 좋은 듯. 다만 조금 더 스프린트를 뛰어야 할 것 같다.
4주 내에 구현이 불가능할 것 같다고 한다면 Polling도 고려해봅시다

**Comment by MoonJeWoong** [comment]
- 스터디 동시 진행 기능에 Sse를 사용하고 발생할 수 있는 문제점에 대한 대비책을 마련할 것인지, 다른 방법을 고려할 것인지
- 만약 다른 방법을 고려한다면 리소스를 들여서 학습하고 고려한 sse를 아예 배재할 것인지
- Sse 자체를 다룰 수 있는 기능을 따로 생각해볼 것인지
- 6차 데모데이에 어떤 Task들을 수행할 것인지, 그것들의 우선순위는 어떻게 산정할 것인지
- 사용자 유치 계획은 언제부터 어떻게 수행할 것인지
